media-gfx/alembic/alembic-1.8.4.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# py311 needs imath-3.1.6+, see PR #28265
```

Comment:
Is this `>=` or `>`?

Reply:
`>=` 3.1.6 is the latest, so `>` isn't currently possible. It's meant prospective for future updates.